pal/minigame/taobao_minigame.ts

```diff
@@ -157,7 +157,7 @@ minigame.stopAccelerometer = function (res: any) {
 // #region SafeArea
 // It should be a value that is not multiplied by dpr
 minigame.getSafeArea = function () {
-    const systemInfo = minigame.getSystemInfoSync();
+    const systemInfo = minigame.getWindowInfoSync();
```

Comment:
this seems not to be a common method for all minigame platforms
please use `my.getWindowInfoSync()`

and add `declare const my: any` before